Cargo.toml

```diff
@@ -132,6 +132,7 @@ members = [
 	"frame/staking",
 	"frame/staking/reward-curve",
 	"frame/staking/reward-fn",
+	"frame/stake-tracker",
```

Comment:
some other name suggestions: 

```
sorted-stakers
sorted-stakers-trackers
sorted-stakers-provider
```

all hinting that under the hood, this pallet provides two implementations of `SortedListProvider` for different staking roles.

Reply:
IMHO `stake-tracker` seems more eloquent. Further implementation details can be deduced from the documentation and the code. But if you feel that it does not give the general idea - can be changed.